docs/package_management.rst

```diff
@@ -0,0 +1,56 @@
+.. _package_management:
+
+Package Management
+==================
+
+.. py:module:: web3.pm
+
+Web3.py provides a number of features for installing and using resources in
+EthPM packages.  These features enable a user to perform any task or realize
+any use-case detailed in the `EthPM specification
+<http://ethpm.github.io/ethpm-spec/index.html>`_.  Below, we'll go over a
+number of examples of how to use this functionality in web3.py.
+
+.. _pm_download_a_package:
+
+Downloading a Package
+---------------------
+
+To use a package with web3.py, it must be downloaded or read from the local
+filesystem.  In the code snippet below, a number of different approaches are
+used to get a package object for a ``greeter`` package:
+
+.. code-block:: python
+
+    >>> from web3 import Web3, HTTPProvider
+    >>> w3 = Web3(HTTPProvider('https://ropsten.infura.io/<token>'))
+
+    >>> # Get a package from a registry
+    >>> greeter_pkg = w3.pm.get_package_from_registry('greeter', registry='ens://packages.ethereum.eth')
+
+    >>> # Get a package from a URI
```

Comment:
Not sure if this is the right place to do it - or in `py-ethpm`'s docs - but a list somewhere of the different support c-a uri's would be useful

Reply:
We could maybe add a note below the code snippet that more c-a URI formats are supported and link to the `py-ethpm` docs.  It seems like there will probably be a section in the `py-ethpm` docs with that information and it seems wrong to duplicate it across the `py-ethpm` and `web3.py` docs.